docker/rails.Dockerfile

```diff
@@ -9,6 +10,12 @@ RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
     curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
     apt-get install -y cmake make nodejs yarn graphicsmagick libvips42
 
+WORKDIR /usr/share/GeoIP
+
+RUN curl "https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country&license_key=${GEOIP_LICENSE_KEY}&suffix=tar.gz" --output geolite2-country.tar.gz && \
+    tar -xvf geolite2-country.tar.gz --strip-components=1 --wildcards '*/GeoLite2-Country.mmdb' && \
+    rm geolite2-country.tar.gz
```

Comment:
The only downside with this being here is that it'll never get update, whereas if it's further down it'll get updated every time we deploy. The size of the db should probably determine this. If it's like 10mb, let's move it down. If it's huge, let's leave it here. Thoughts?

Reply:
The size is 5.3MB. I'll move it down then.